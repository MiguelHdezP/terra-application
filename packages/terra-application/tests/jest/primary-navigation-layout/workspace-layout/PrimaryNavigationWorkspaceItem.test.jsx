import React from 'react';
import PrimaryNavigationWorkspaceItem from '../../../../src/primary-navigation-layout/workspace-layout/PrimaryNavigationWorkspaceItem';

describe('ActionMenuHeader', () => {
  it('should render with minimal props', () => {
    const wrapper = shallow((
      <PrimaryNavigationWorkspaceItem
        itemKey="test key"
        label="test label"
      />
    ));
    expect(wrapper).toMatchSnapshot();
  });

  it('should render with all props', () => {
    const wrapper = shallow((
      <PrimaryNavigationWorkspaceItem
        itemKey="test-key"
        label="test label"
        metaData={{ key: 'test-key' }}
        render={() => {}}
      />
    ));
    expect(wrapper).toMatchSnapshot();
  });
});
