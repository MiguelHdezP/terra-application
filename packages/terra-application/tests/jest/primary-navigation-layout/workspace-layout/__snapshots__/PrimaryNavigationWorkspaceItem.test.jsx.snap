// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionMenuHeader should render with all props 1`] = `<div />`;

exports[`ActionMenuHeader should render with minimal props 1`] = `<div />`;
